atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -123,11 +124,13 @@ def __init__(self, tasker, workflow,
         else:
             self.log.info("Koji task ID: %s", self.koji_task_id)
 
-        try:
+        self.koji_build_id = self.workflow.exit_results.get(KojiImportPlugin.key)
+        if self.koji_build_id is None:
```

Comment:
I think i want if not, because there's a possibility that self.workflow.exit_results.get(KojiImportPlugin.key) might not return anything because arrangement{1,2} calls don't even have a koji_import plugin.

The logic should be get a result from koji_import which might not be valid. if it isn't valid, check koji_promote. Only if both give an invalid result should we believe that there's no koji_build_id.